modules/stochastic_tools/include/reporters/ActiveLearningGPDecision.h

```diff
@@ -76,6 +76,11 @@ class ActiveLearningGPDecision : public ActiveLearningReporterTempl<Real>,
   /// The learning function parameter
   const Real & _learning_function_parameter;
 
+  /// Store all the input vectors used for training
+  std::vector<std::vector<Real>> _inputs_batch;
+  /// Store all the outputs used for training
+  std::vector<Real> _outputs_batch;
+
```

Comment:
Why did you need to move these?

Reply:
The derived `BiFidelityActiveLearningGPDecision` class could access the private variables.